app/controllers/corrections/UndeclaredCountryCorrectionController.scala

```diff
@@ -67,7 +67,6 @@ class UndeclaredCountryCorrectionController @Inject()(
             value =>
               for {
                 updatedAnswers <- Future.fromTry(request.userAnswers.set(UndeclaredCountryCorrectionPage(periodIndex, countryIndex), value))
-                _ <- cc.sessionRepository.set(updatedAnswers)
```

Comment:
why does this get removed?

Reply:
We don't think we need this saved, it's not used from what we've seen and just made it easier for cleanup. Unless you feel we should be saving this?